README.md

```diff
@@ -0,0 +1,71 @@
+
+# EMC_post
+
+The EMC_post package provides tools for post processing for NCEP models.
+
+This is part of the [NCEPLIBS](https://github.com/NOAA-EMC/NCEPLIBS)
```

Comment:
It also serves as a standalone package that is distributed to the community. And is built outside of nceplibs for some UFS applications (e.g. SRW v1.0.0).  Should clarify this somehow?

Reply:
I have clarified.